routes/reminders.js

```diff
@@ -11,8 +11,22 @@ function removeFamilyProperty(item) {
 }
 
 router.get('/', (req, res, next) => {
-  reminders.index(req.user.family, req.query.start, req.query.limit)
-  .then(rows => {
+  const start = parseInt(req.query.start);
+  let limit = parseInt(req.query.limit);
```

Comment:
currently, never :) the checks in the DAO are only there in case of future programming errors.

The NaN cases are handled by the condtions "if (Number.isNaN(limit))" and "if (Number.isNaN(start))" below.